packages/openid4vc/tests/openid4vc.e2e.test.ts

```diff
@@ -218,6 +223,7 @@ describe('OpenId4Vc', () => {
       },
     })
     const openIdIssuerTenant2 = await issuerTenant2.modules.openId4VcIssuer.createIssuer({
+      dPoPSigningAlgValuesSupported: ['EdDSA'],
```

Comment:
Thinking in general whether this is something you want to configure per issuer? It will make uodates in multi-tenancy quite cumbersome. But i also see the case for having different values per issuer